tests/onnxruntime/test_modeling.py

```diff
@@ -1386,8 +1386,10 @@ def test_pipeline_on_gpu(self, model_arch):
         # check model device
         self.assertEqual(pipe.model.device.type.lower(), "cuda")
         # compare model output class
-        self.assertTrue(isinstance(outputs[0]["generated_text"], str))
-        self.assertTrue(len(outputs[0]["generated_text"]) > len(text))
+        self.assertTrue(isinstance(outputs[0]["translation_text"], str))
+
+        # does not pass with hf-internal-testing/tiny-random-t5 , but OK with t5-base
+        # self.assertTrue(len(outputs[0]["translation_text"]) > len(text))
```

Comment:
why is not passing?

Reply:
no idea, it's not passing either with `hf-internal-testing/tiny-random-t5` + transformer model

Reply:
The test was not run up to know due to the require gpu decorator

Reply:
can you check why? like whats the issue?

Reply:
The output `translation_text` is `""`. I just think it's the model being bad.

Reply:
even when setting explicitly `min_length`?

Reply:
When the pipeline parameter  `return_tensors` is `False`, the padding tokens are removed from the output. Hence the issue. Fixed now.